app/models/user.rb

```diff
@@ -27,6 +27,34 @@ def scoped_service_providers
       sort_by! { |sp| sp.friendly_name.downcase }
   end
 
+  def user_deletion_history
+    PaperTrail::Version
+      .where(event: 'destroy', item_type: 'UserTeam')
+      .where("object ->>'user_id' = '?'", id)
+  end
+
+  def user_deletion_report_item(record)
+    {
+      user_id: record[0]['user_id'],
+      user_email: User.find_by(id: record[0]['user_id'])&.email,
+      team_id: record[0]['group_id'],
+      team_name: Team.find_by(id: record[0]['group_id'])&.name,
+      removed_at: record[1],
+      whodunnit_id: record[2],
+      whodunnit_email: User.find_by(id: record[2])&.email
```

Comment:
can we destructure `record` to make it clearer what the fields are? I can't tell what the `record[0]` is so we need a better name there obvi

```suggestion
  def user_deletion_report_item(record)
    something, removed_at, whodunnit_id = record
    {
      user_id: something['user_id'],
      user_email: User.find_by(id: something['user_id'])&.email,
      team_id: something['group_id'],
      team_name: Team.find_by(id: something['group_id'])&.name,
      removed_at: removed_at,
      whodunnit_id: whodunnit_id,
      whodunnit_email: User.find_by(id: whodunnit_id)&.email
```